utils/twcustom.py

```diff
@@ -0,0 +1,61 @@
+
+from twilio.twiml.voice_response import VoiceResponse
+
+
+class CustomVoiceResponse(VoiceResponse):
+    voice = "Polly.Arthur-Neural"
```

Comment:
excuse me, 
Can we use a young female British voice?

![image](https://github.com/Chugaister/scoundef/assets/32039752/71ea739d-59c3-4fc7-9f18-6a8c4c63c89d)